src/ConfigFactory/ConfigFactory.cs

```diff
@@ -37,15 +37,22 @@ public static ConfigPageModel Build(IConfigModule module)
     /// <inheritdoc cref="Append{T}(ConfigPageModel)"/>
     public static ConfigPageModel Append(this ConfigPageModel configPageModel, IConfigModule module)
     {
+        //Initialize Save handler
         configPageModel.PrimaryButtonEvent += () => {
             module.Save();
             return Task.CompletedTask;
         };
+        //Initialize Cancel handler
+        configPageModel.SecondaryButtonEvent += () =>
+        {
+            module.Reset();
+            return Task.CompletedTask;
+        };
 
-        foreach ((_, (PropertyInfo info, ConfigAttribute attribute)) in module.Properties) {
-            object? value = info.GetValue(module, null);
-            if (_builders.FirstOrDefault(x => x.IsValid(info.PropertyType)) is IControlBuilder builder) {
-                ConfigGroup group = GetConfigGroup(configPageModel, module, attribute);
+        foreach ((_, (var info, var attribute)) in module.Properties) {
+            var value = info.GetValue(module, null);
+            if (_builders.FirstOrDefault(x => x.IsValid(info.PropertyType)) is { } builder) {
+                var group = GetConfigGroup(configPageModel, module, attribute);
```

Comment:
Prefer explicit type names and `is not null`

```suggestion
        foreach ((_, (PropertyInfo info, ConfigAttribute attribute)) in module.Properties) {
            object? value = info.GetValue(module, null);
            if (_builders.FirstOrDefault(x => x.IsValid(info.PropertyType)) is not null) {
                ConfigGroup group = GetConfigGroup(configPageModel, module, attribute);
```

Reply:
Okay.